### DISCUSSION TITLE
svg font trace

### DISCUSSION THREAD

**Discussion by btrem** [edent/SuperTinyIcons#779] [Q&A] [answered]
Anyone know of a utility to provide an svg trace of letters in a specific font? I'm working on a logo that is very simple: 3 letters in 3 adjacent boxes. But the `path` elements were generated by a png-to-svg utility, and the `path d` attributes are very long and complex. I'm trying to simplify them by editing the `d` attributes, but it's really tedious, hence my query.

**Comment by edent** [comment]
You can convert the font to SVG using something like https://fontsvg.com/

You can simplify SVGs using https://svgomg.net/

**Comment by btrem** [comment]
Thanks for the link. Unfortunately, it does not offer the font I need (Trebuchet bold). Still, might be helpful in the future.

**Comment by Eiim** [comment, marked as answer]
If you have the font on your computer, you can type the letters in Inkscape and convert text to path from there. Then you can optimize with Inkscape and/or the link @edent provided (Inkscape's algorithm generally isn't as good so I'd usually prefer SVGOMG).

**Reply by btrem** [reply]
Yeah, that looks like it will work. My Ubuntu version of the Trebuchet font ("Trebuchet MS"), is, sadly, a tiny bit different than the one the logo uses, but I can try to tweak the paths. Thanks for the tip.

Aside: Github's UI for discussions is not great. I kept trying to click the gray arrow circle to mark this answer as correct. It took me several tries, over a period of time, before I realized the *text* next to the arrow is the button, not the arrow itself. Shame, GH, shame!
